tests/integration/rest.integration.test.ts

```diff
@@ -0,0 +1,171 @@
+export {};
+const fetch = require('node-fetch');
+import { TOKEN_GEN_TYPES, TOKEN_GEN_DATA, generateJWT } from './testUtilities/integrationTestUtils';
+
+describe('HiFi API REST Calls', () => {
+	let token: string;
+
+    beforeAll(async () => {
```

Comment:
fix indentation please (not just here, but everywhere - vs code's format document should work)